src/sentry/tasks/commits.py

```diff
@@ -93,10 +93,16 @@ def fetch_commits(release_id, user_id, refs, prev_release_id=None, **kwargs):
             )
             continue
 
-        try:
-            provider_cls = bindings.get('repository.provider').get(repo.provider)
-        except KeyError:
-            continue
+        if repo.provider.startswith('integrations:'):
+            try:
+                provider_cls = bindings.get('integration-repository.provider').get(repo.provider)
+            except KeyError:
+                continue
+        else:
+            try:
+                provider_cls = bindings.get('repository.provider').get(repo.provider)
+            except KeyError:
+                continue
```

Comment:
nit, you could do something like:
```python
binding_key = 'integration-repository.provider' if repo.provider and repo.provider.startswith('integrations:') else 'repository.provider'
```
and then do `bindings.get(binding_key)...`

or somethign like that to avoid the duplicate try/except